src/Home.js

```diff
@@ -84,26 +186,41 @@ class Home extends Component {
         </div>
       );
     }
+
     return (
-      <Main id={HOME_MAIN_ID} showSearch>
-        <SpaceGrid spaces={filteredSpaces} showActions saved />
+      <Main showSearch handleOnSearch={this.handleOnSearch} id={HOME_MAIN_ID}>
+        {noSpacesAvailable ? (
+          <NoSpacesAvailable />
+        ) : (
+          <div className={classes.home}>
+            {this.renderRecentSpaces()}
+            <br />
```

Comment:
Maybe add a `Divider` component here?

Reply:
From Material UI

Reply:
I changed to it

![image](https://user-images.githubusercontent.com/11229627/81657653-54341400-9438-11ea-95af-2a629df46778.png)